aws-lc-rs/src/tlsprf/tests/fips.rs

```diff
@@ -0,0 +1,64 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0 OR ISC
+
+#![cfg(debug_assertions)]
+
+use crate::fips::{assert_fips_status_indicator, FipsServiceStatus};
+use crate::tlsprf::{Secret, P_SHA256, P_SHA384, P_SHA512};
+
+macro_rules! prf {
+    ($name:ident, $alg:expr, $size:expr, $label:expr, $expect:path) => {
+        #[test]
+        fn $name() {
+            let secret = Secret::new($alg, &[42u8; $size]).expect("secret created");
```

Comment:
Is there any particular reason why `42u8` is used here?

Reply:
Arbitrary with no meaning, just needed a value for testing